docs/index.md

```diff
@@ -211,7 +211,7 @@ Let us take a look at few recommendations on how and where to run the chaos test
 #### Chaos testing in Practice
 
 ##### OpenShift organization
```

Comment:
Should we leave this openshift refernce since its focusing on openshift or move the whole section? 

Is tekton a kubernetes ci, or specific to openshift?

Reply:
I think we should leave it as it's more of a story around how the chaos testing is being practiced in the OpenShift organization. Thoughts?

Tekton can run both on Kubernetes and OpenShift.

Reply:
agree, think on line 214 we should keep the OpenShift reference then

Reply:
Updated, thanks.